pywbem/__init__.py

```diff
@@ -64,8 +64,7 @@
 _python_m = sys.version_info[0]  # pylint: disable=invalid-name
 _python_n = sys.version_info[1]  # pylint: disable=invalid-name
 
-# !!! Make sure to keep the supported Python versions in sync
-# between setup.py, setup.cfg and pywbem/__init__.py !!!
+# Keep the checked Python versions in sync with setup.py
```

Comment:
Not shure what the word checked means in this context.

Reply:
The code below verifies the supported Python versions. Any suggestio s for a better wording?

Reply:
DONE - changed to "Keep these Python versions in sync with setup.py"